config/routes.rb

```diff
@@ -1,3 +1,6 @@
 # frozen_string_literal: true
 Rails.application.routes.draw do
+  resources :buildings
+  resources :suites
+  resources :rooms
```

Comment:
I think that we *might* want rooms outside of suites sometimes; what if someone wants to filter by largest bedrooms or something?